### DISCUSSION TITLE
Qwik framework support?

### DISCUSSION THREAD

**Discussion by LazyClicks** [chakra-ui/ark#1340] [Ideas]
### Description

can we get ark-ui support for qwik please?

### Proposed Solution or API

support for qwik i guess?

**Comment by cschroeter** [comment]
Thanks for taking your time to create this issue. This has already been discussed in https://github.com/chakra-ui/zag/discussions/392

To my knowledge, technical it is possible but it would create a lot of code duplication in Zag.js because of this `event.preventDefault` limitations in qwik. 

@LazyClicks The workaround you posted looks interesting. If you can come up with a PoC that would be huge.

**Comment by gabrielgrant** [comment]
beyond the `useVisibleTask` workaround, there's now real a `preventDefault` fix that is currently being working on in qwik: https://github.com/BuilderIO/qwik/issues/4496#issuecomment-1775426909

Does that seem like the only blocker to a Qwik ark implementation?

**Comment by plandem** [comment]
any updates?

**Reply by LazyClicks** [reply]
i assume it's not likely to happen, take a look into qwik ui instead

**Reply by plandem** [reply]
at first sight, qwik ui - quite basic that can be done via vanilla-extract/pandacss

**Reply by LazyClicks** [reply]
> at first sight, qwik ui - quite basic that can be done via vanilla-extract/pandacss

except that u would need to work on the accessibility of the components urself, qwik ui does it for u
